install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-xenial

```diff
@@ -1,3 +1,9 @@
+securedrop-app-code (1.2.1+xenial) xenial; urgency=medium
+
+  * See changelog.md 
+
+ -- SecureDrop Team <securedrop@freedom.press>  Wed, 19 Feb 2020 14:40:43 +0000
+
```

Comment:
ah we should remove the below entry (`securedrop-app-code (1.2.1~rc1+xenial)`) - that is the rc